libs/pyTermTk/TermTk/TTkWidgets/TTkTerminal/terminal_screen.py

```diff
@@ -1,4 +1,4 @@
-    # MIT License
+1    # MIT License
```

Comment:
err

Reply:
fuck, this is my working Fucking MAC that trigger 1s without pressing the key.

Pushed the fix to main without PR